infrastructure/lib/kit-infrastructure.ts

```diff
@@ -8,7 +8,7 @@ import { AWSFluentBit } from './addons/fluent-bit-for-aws'
 import { FluxV2 } from './addons/fluxv2'
 import { Karpenter } from './addons/karpenter'
 import { KIT } from './addons/kit'
-import { KubectlLayer } from 'aws-cdk-lib/lambda-layer-kubectl'
+import { KubectlV26Layer } from '@aws-cdk/lambda-layer-kubectl-v26'
```

Comment:
why 26 ? can't we on the default and make sure our stack is upto date ?

Reply:
the default kubectl was still old and the stack creation was failing because of some resource mismatch. I think this is a better way because we can control both the kubectl version and kubernetes cluster version now.